### PR TITLE
Feature/#457 need to show read only paths to folder locations in settings

### PR DESCRIPTION
**Pull request by Rahul139-stack**
This Pr includes:

- Able to get Install app path
- Able to app logs path.
- 
image

### CODE REVIEW

File: electron/app/main/index.ts [hunk 1]
```diff
@@ -396,6 +396,16 @@ function addIpcEvents(window: BrowserWindow) {
       logger.log(`ipc getServices`)
       return serviceManager.getServicesSimple()
     },
+    getAppPath(): any {
+      const appPath = app.getPath("appData") + `\\TypeRefinery`
```
Comment (Harikarthyk) [inline electron/app/main/index.ts:400]: will this path which you have mentioned will work for all the OS ?

**Review by Harikarthyk — CHANGES_REQUESTED** [review]
Can you please update the UI 

Input can be disabled as default.
Can have some margins Font weight can be reduced and size increased. 

 Copy Icon can be added new the title, it can have the app primary color.

**Comment by Rahul139-stack** [comment]
> Can you please update the UI?
> 
> Input can be disabled by default. Can have some margins The font weight can be reduced and the size increased.
> 
> A copy icon can be added to the title; it can have the app's primary colour.
> 
> Along with this, please provide an option to open the file by clicking a button.

Updated the UI as required

**Comment by Harikarthyk** [comment]
Please add the Preview of how it looks.

**Comment by Rahul139-stack** [comment]
> Please add the Preview of how it looks.

image

**Review by Harikarthyk — APPROVED** [review]

**Pull request merged**